src/components/NodeChart/index.tsx

```diff
@@ -53,16 +51,38 @@ const NodeChart: FC<Props> = ({ label, centerCount, data }: Props) => {
   const [hoveredSlice, setHoveredSlice] = useState<PieSectorDataItem>(null);
 
   const dataset: PieSectorDataItem[] = useMemo(() => data.filter(({ value }) => value > 0), [data]);
-  const onMouseOver = useCallback((data) => setHoveredSlice(data), []);
-  const onMouseLeave = useCallback(() => setHoveredSlice(null), []);
+
   const showDefaultCenter: boolean = useMemo(
     () => (dataset.length === 0 && hoveredSlice === null) || hoveredSlice?.value === 0,
     [dataset, hoveredSlice]
   );
 
+  const reformattedLabel = useMemo<string>(() => {
+    const replacedLabel = label?.replace(/_/g, " ") || "";
+
+    // If the label has no spaces, capitalize the first letter to avoid
+    // titleCase from performing a full title case conversion
+    if (replacedLabel?.indexOf(" ") === -1) {
```

Comment:
Just some additional context:

`titleCase` forces all characters to lowercase first before applying uppercasing to the words. 

This works fine when a node name has multiple words (Example: `Principal Investigator` or `Non Targeted Therapy`) but it's pretty bad for the new nodes that CDS just created (Example: `NonDICOMCTimages` which would become `Nondicomctimages`).

My solution ATM is just to short-circuit the reformatting when there are no spaces. We never really had requirements that dictated how we format node labels, so it's somewhat inconsistent across the board.